hwy/tests/combine_test.cc

```diff
@@ -284,6 +284,151 @@ HWY_NOINLINE void TestAllConcatOddEven() {
   ForAllTypes(ForShrinkableVectors<TestConcatOddEven>());
 }
 
+class TestTruncatingResizeBitCast {
+#if HWY_TARGET != HWY_SCALAR
+ private:
+  template <class DTo, class DFrom>
+  static HWY_INLINE void DoTruncResizeBitCastTest(DTo d_to, DFrom d_from) {
+    const VFromD<DFrom> v = Iota(d_from, 1);
+    const VFromD<DTo> expected = Iota(d_to, 1);
+
+    const VFromD<DTo> actual_1 = ResizeBitCast(d_to, v);
+    HWY_ASSERT_VEC_EQ(d_to, expected, actual_1);
+
+    const VFromD<DTo> actual_2 = ZeroExtendResizeBitCast(d_to, d_from, v);
+    HWY_ASSERT_VEC_EQ(d_to, expected, actual_2);
+  }
+#endif  // HWY_TARGET != HWY_SCALAR
+ public:
+  template <class T, class D>
+  HWY_NOINLINE void operator()(T /*unused*/, D d) {
```

Comment:
@johnplatts our CI is failing because `d` is unused #if HWY_TARGET == HWY_SCALAR. Would you mind adding (for example) a `(void)d` here and in line 392?